src/index.html

```diff
@@ -6,13 +6,76 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Product cards</title>
+    <link
+      rel="stylesheet"
+      href="./styles/card.scss"
```

Comment:
You are linking a .scss file here, but browsers can't interpret SCSS. You should compile your SCSS to CSS, and link the CSS file in your HTML.